modules/apps/27-interchain-accounts/controller/keeper/handshake_test.go

```diff
@@ -37,7 +38,7 @@ func (suite *KeeperTestSuite) TestOnChanOpenInit() {
 			false,
 		},
 		{
-			"invalid port ID",
+			"invalid port",
```

Comment:
mega nit: makes more sense to me with the message `invalid portID`. Both vars declared below use this naming